test_infra/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "awswrangler-test-infrastructure"
-version = "3.12.0"
+version = "3.12.1"
 description = "CDK test infrastructure for AWS SDK for pandas"
 authors = ["Amazon Web Services"]
```

Comment:
noticed this is not per [spec](https://packaging.python.org/en/latest/specifications/pyproject-toml/#authors-maintainers)